java-manta-it/src/test/java/com/joyent/manta/client/multipart/EncryptedServerSideMultipartManagerSerializationIT.java

```diff
@@ -36,40 +36,51 @@
 
 import static org.testng.Assert.fail;
 
-@Test(groups = { "encrypted" })
+/**
+ * Tests for verifying the behavior of {@link com.joyent.manta.serialization.EncryptedMultipartSerializer}
+ * and {@link EncryptedServerSideMultipartManager} with {@link MantaClient}.
+ *
+ * @author <a href="https://github.com/dekobon">Elijah Zupancic</a>
+ * @author <a href="https://github.com/nairashwin952013">Ashwin A Nair</a>
+ */
+@Test(groups = {"encryptable", "multipart"})
```

Comment:
Refactored groups that were initially defined as `encrypted` to `encryptable` so as to make a clear distinction between names of groups since we have a new test group defined as `unencrypted` [here](https://github.com/joyent/java-manta/pull/507/files#diff-dd127e7993afbd9ec23c3cfc2828d155R70) that comprises of all tests verifying operations in the SDK which **don't utilize** encryption.

Reply:
My objection to the word "encryptable" is that the word sounds a bit unnatural. Why is it preferred over "encrypted"?

Reply:
Will be refactoring `encryptable` -> `encrypted`

Reply:
Review noted, will come up with a solution.

Reply:
We might have to use `beanshell` scripts which will defined in the `testing-it.xml` file.

References: 
- [Test-NG Docs](https://testng.org/doc/documentation-main.html#beanshell).
- Blog [post](https://rationaleemotions.wordpress.com/2013/08/21/beanshell-and-testng/) that I found online.